osu.Game/Overlays/Comments/VotePill.cs

```diff
@@ -69,6 +70,8 @@ protected override void LoadComplete()
             votesCount.Value = comment.VotesCount;
             isVoted.BindValueChanged(voted => background.Colour = voted.NewValue ? AccentColour : OsuColour.Gray(0.05f), true);
             votesCount.BindValueChanged(count => votesCounter.Text = $"+{count.NewValue}", true);
+
+            api.LocalUser.BindValueChanged(user => disabled = user.NewValue?.Id == comment.UserId ? true : false, true);
```

Comment:
I can't imagine how `? true : false` is meanful.

Reply:
oopsie